shopfloor_checkout_package_measurement/actions/schema.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from odoo.addons.component.core import Component
+
+
+class ShopfloorSchemaAction(Component):
+    _inherit = "shopfloor.schema.action"
+
+    def package(self, with_packaging=False):
+        res = super().package(with_packaging=with_packaging)
+        res.update(
```

Comment:
doesn't make sense only if `with_packaging` is True?

Reply:
I would say no, because the measurements are for packages and not for packaging.